Vagrantfile.example

```diff
@@ -42,9 +42,20 @@ Vagrant.configure("2") do |config|
     default.vm.provision "shell", path: "build_scripts/web_setup/web_setup.sh", args: 'single_server'
     default.vm.provision "shell", path: "build_scripts/app_setup/app_setup.sh"
     default.vm.provision "shell", path: "build_scripts/es_setup/es_setup.sh", args: 'single_server'
-    default.vm.provision "shell", path: "build_scripts/db_setup/db_setup.sh"
+    default.vm.provision "shell", path: "build_scripts/db_setup/db_setup.sh", args: 'single_server'
```

Comment:
What was this for? Just wondering.

Reply:
This is so that when we configure postgres in our "single server" vm, we do not use the configuration files in the db_setup directory (which contain replication variables) but rather use the ones in default directory. The argument is checked in line 31 of `build_scripts/db_setup/db_setup.sh`.